spec/lib/vmdb/loggers_spec.rb

```diff
@@ -0,0 +1,52 @@
+describe Vmdb::Loggers do
+  let(:log_file) { Rails.root.join("log", "foo.log").to_s }
+  describe "#create_multicast_logger" do
```

Comment:
Minor style...please add blank lines around describes to separate them

Reply:
Also, minor, but since this is a private method, can you clarify that in the describe line?  It makes it clearer why you have to do .send below.  I usually do something like `describe "#create_multicast_logger (private)" do`

Reply:
Sweet, all for this, just assumed rubocop would have yelled at me (feel like it has in the past).

Reply:
Yeah, I expected rubocop to yell about it too, so I was surprised